pkg/api/handlers/types.go

```diff
@@ -271,6 +271,15 @@ func ImageDataToImageInspect(ctx context.Context, l *libpodImage.Image) (*ImageI
 	if err != nil {
 		return nil, err
 	}
+
+	rootfs := docker.RootFS{}
+	if info.RootFS != nil && info.RootFS.Layers != nil {
```

Comment:
Should this be split?  i.e. test RootFS for nil and then set RootFS.Type if it is not nil, then test for RootFS.Layers and set that if it is not nil.

Reply:
I fix that. It turned out that it's OK to call `len()` and `range` on nil slice.